main.nf

```diff
@@ -233,7 +234,7 @@ process call_variants_individual {
         tuple strain, path("${region}.g.vcf.gz")
 
     """
-        gatk HaplotypeCaller --java-options "-Xmx${task.memory.toGiga()}g -Xms1g" \\
+        gatk HaplotypeCaller --java-options "-Xmx${task.memory.toGiga()}g -Xms1g -XX:ConcGCThreads=${task.cpus}" \\
```

Comment:
Why are you allocating additional threads for garbage collection? Does this improve performance?

Reply:
i think it's from here: https://gatk.broadinstitute.org/hc/en-us/articles/360035532372-Java-is-using-too-many-resources-threads-memory-or-CPU-
gatk4 does not have built-in multi threading, they moved all that functionality to depend on spark. so it actually does not help too much to allocate gatk multiple cores. but, java has threads of its own, so this is telling java not to use more than there is available. i forgot whether this to deal with an error or warning.